### ISSUE TITLE
Control Center: Add Blocking Stats

### ISSUE THREAD

**Issue by winsmith** [cliqz/user-agent-ios#147]
Blocked Entities are ads, trackers, etc blocked by the browser. For now, we're continuing to use Mozilla's blocking tech, so this is just displaying the existing numbers:

- [ ] Research the API to Mozilla's blocking mechanism and see how and what info to extract
- [ ] Show the number of blocked entities in Control Center
- [ ] Allow the user to toggle all blocking on and off in Control Center
- [ ] Show the number and types of blocked entities in a sub menu of the Control Center
- [ ] Badge the Control Center Icon with the number of entities blocked

See also #147

**Comment by chrmod** [comment]
closing in favor of https://github.com/cliqz/user-agent-ios/issues/404
